ui/templates/index.html

```diff
@@ -32,7 +32,7 @@ <h1 class="display-4">Asset Manager</h1>
                     {% endfor %}
                 </select>
             </div>
-            <button type="submit" class="btn btn-primary mb-2" style="margin-top: 10px">Enter</button>
+            <button type="submit" class="btn btn-primary mb-2" style="margin-top: 10px">List projects in store</button>
```

Comment:
This is quite a long text for the label of a button. Did you want to use an icon + tooltip instead?